### DISCUSSION TITLE
x509: certificate signed by unknown authority - in offline managed installation

### DISCUSSION THREAD

**Discussion by Josef-t11** [Dynatrace/dynatrace-configuration-as-code#291] [Archive] [answered]
One of our customers (government agency) is using offline managed cluster which we are accessing via customer VPN without access to internet.
I tried to get configuration changes via monaco **download** (ver 1.5.0) and got the message "x509: certificate signed by unknown authority". Is it possible to add "non check certificate" behavior - e.g. via new parameter ?

BR, Josef Solnicky, TietoEvry

**Comment by tatomir146** [comment, marked as answer]
Hi @Josef-t11, monaco is using host system trust store to determine if SSL is trusted. If you add self-signed certificate to the hosts trust store, monaco will accept it.

**Reply by tatomir146** [reply]
We don't plan to add self-signed certificate support to monaco.

**Reply by tpnoonan** [reply]
I have added our self signed certificate (pem) to the trust store and then updated and verified that the certificate was added along with rebooting the server and I'm still receiving ultimately a "...failed to verify certificate..." from monaco.  

I'm using the Linux X64 latest version.

**Reply by arthurpitman** [reply]
Hi @tpnoonan, as most likely this is dependent on the exact network setup, it would be best to create a [support archive ](https://docs.dynatrace.com/docs/shortlink/configuration-as-code-logging#support-archive) and reach out to Dynatrace Support to work through this.

**Reply by tpnoonan** [reply]
Thank you @arthurpitman, I will do so.

**Reply by dcryans** [reply]
@tpnoonan 
Not sure if you found your answer since March

On Ubuntu running in WSL2, I had to specify this:
export SSL_CERT_DIR=/usr/local/share/ca-certificates/

Golang wouldn't use my installed certs otherwise
